package.json

```diff
@@ -39,11 +39,13 @@
     "babel-plugin-add-module-exports": "^0.1.2",
     "babel-preset-es2015": "^6.5.0",
     "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-2": "^6.18.0",
```

Comment:
I just moved it from dependency to devDependency, it's definitely not a dependency.